editorial/topic-slate-2-demand-first.html

```
@@ -0,0 +1,230 @@
+<title>Google-User-Wanna-Know Slate</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)